src/types/redis_bloom_chain.h

```diff
@@ -30,20 +30,38 @@ const uint32_t kBFDefaultInitCapacity = 100;
 const double kBFDefaultErrorRate = 0.01;
 const uint16_t kBFDefaultExpansion = 2;
 
+enum class BloomInfoType {
+  ALL,
```

Comment:
You can format enum values as `kAll, kCapacity, etc` (see here: https://google.github.io/styleguide/cppguide.html#Enumerator_Names)